index.js

```diff
@@ -716,7 +716,7 @@
     ('HtmlElement')
     ([])
     (function(x) {
-       return /^\[object HTML.+Element\]$/.test (toString.call (x));
+       return /^\[object HTML.*Element\]$/.test (toString.call (x));
```

Comment:
How about disallow space here? `^\[object HTML[^\s]*Element\]$`

Reply:
@Avaq What do you think about changing the regex to `^\[object HTML[^\s]*Element\]$` in order to disallow spaces after `HTML`?

So `HTML ElementDiv` would fail while `HTML.ElementDiv` would be allowed.

Reply:
We will accept infinitely many invalid strings in either case; I'm inclined to stick with the simpler `.*`.

By the way, `\S` is shorthand for `[^\s]`. :)

Reply:
> We will accept infinitely many invalid strings in either case

That was my thinking too.

Reply:
I guess, since `HTMLElement` is an exotic object, baking in too many assumptions is bound to fail in edge cases.

The real test-case would be `return x instanceof HTMLElement` of course, but I see how that is difficult to test in an environment without `HTMLElement`, like nodejs.

Sanctuary-def's test of `HTMLElement` is good enough so that Sanctuary works client-side without requiring users to create a custom `HTMLElement` definition.